client/resource_group/controller/controller.go

```diff
@@ -472,6 +475,28 @@ func (c *ResourceGroupsController) OnResponse(
 	return tmp.(*groupCostController).onResponse(req, resp)
 }
 
+// IsBackgroundRequest If the resource group has background jobs, we should not record consumption and wait for it.
+func (c *ResourceGroupsController) IsBackgroundRequest(ctx context.Context,
+	resourceGroupName, requestResource string) bool {
+	gc, err := c.tryGetResourceGroup(ctx, resourceGroupName)
+	if err != nil {
+		failedRequestCounter.WithLabelValues(resourceGroupName).Inc()
+		return false
+	}
+
+	gc.metaLock.Lock()
+	defer gc.metaLock.Unlock()
+	if bg := gc.meta.BackgroundSettings; bg != nil {
+		if len(requestResource) == 0 || len(bg.JobTypes) == 0 {
+			return false
+		}
+		if idx := strings.LastIndex(requestResource, "_"); idx != -1 {
```

Comment:
Should also implement the fallback to default logic here

Reply:
Because of this pr https://github.com/tikv/client-go/pull/890, I think currently it's work fine...Any scenarios I haven't considered?

Reply:
https://github.com/tikv/tikv/pull/15100/files#diff-4fc0c7882a408c1e08b80a597c1d551dd92301f7f6da911b5993e41ac82a265eR244 You mean just like that?

Reply:
Yes. That means if a non-default resource group's config is not explicitly set, you should uses the default group's rule to check it.